src/main/java/gregtech/integration/hwyla/provider/RecipeLogicDataProvider.java

```diff
@@ -64,10 +66,11 @@ public List<String> getWailaBody(ItemStack itemStack, List<String> tooltip, IWai
                 String endText = null;
 
                 if (accessor.getTileEntity() instanceof IGregTechTileEntity gtte) {
-                    if (gtte.getMetaTileEntity() instanceof SteamMetaTileEntity || gtte.getMetaTileEntity() instanceof MetaTileEntityLargeBoiler) {
-                        endText = ": " + absEUt + TextFormatting.RESET + " L/t " + I18n.format("gregtech.top.steam")+ " " + I18n.format(Materials.Steam.getUnlocalizedName());
+                    MetaTileEntity mte = gtte.getMetaTileEntity();
+                    if (mte instanceof SteamMetaTileEntity || mte instanceof MetaTileEntityLargeBoiler || mte instanceof RecipeMapSteamMultiblockController) {
+                        endText = ": " + absEUt + TextFormatting.RESET + " L/t " + I18n.format("gregtech.top.of") + I18n.format(Materials.Steam.getUnlocalizedName());
```

Comment:
Was it meant to remove the space here?

Reply:
I removed the space, so that if the lang is empty, there's not a double space (unless something else already handles that)

Reply:
There should not be any reason that the lang would be empty. And even if the lang was empty, people would not be able to see the space. So it should probably just be kept in.

Reply:
alright